README.markdown

```diff
@@ -2,7 +2,17 @@
 Github-Viewer.slack
 ======================
 
-LICENSE
----------
+# How to setup in your slack team
+1. Create a bot user at [here](https://api.slack.com/apps?new_app=1)
+2. Get OAuth API Token of the bot user <a name="api_token">***!***</a>
 
-MIT
+# How to setup in the local
+1. `virtualenv .venv && . .venv/bin/activate` (if you need it)
```

Comment:
This is what the user considers(Ths user may use pyenv e.t.c.).

Please delete this role.